encode/src/main/scala/org/broadinstitute/monster/etl/encode/extract/EncodeIngestExtract.scala

```diff
@@ -0,0 +1,182 @@
+package org.broadinstitute.monster.etl.encode.extract
+
+import org.broadinstitute.monster.etl.encode.transforms._
+import caseapp.HelpMessage
+import com.spotify.scio.ContextAndArgs
+import org.broadinstitute.monster.etl.encode.EncodeEntity
+import com.spotify.scio.extra.json._
+import com.spotify.scio.values.SCollection
+import io.circe.JsonObject
+import io.circe.syntax._
+
+/**
+  * Extract raw metadata from ENCODE
+  */
+object EncodeIngestExtract {
```

Comment:
I'd rename this to something like `ExtractionPipeline`

Reply:
(with the intent that we'll rename the other one to `TransformationPipeline` at some point)